code/clue.js

```diff
@@ -1,59 +1,408 @@
+  color: "DarkCyan"
+};
+
+const lounge = {
+  name: "Lounge",
+  image: "assets/lounge.svg",
+  color: "DarkGoldenRod"
+};
+
+const ballroom = {
+  name: "Ballroom",
+  image: "assets/chandelier.svg",
+  color: "DarkSlateBlue"
+};
+
+const hall = {
+  name: "Hall",
+  image: "assets/hanger.svg",
+  color: "Khaki"
+};
+
+const spa = {
+  name: "Spa",
+  image: "assets/beauty.svg",
+  color: "HotPink"
+};
+
+const livingRoom = {
+  name: "Living Room",
+  image: "assets/lamp.svg",
+  color: "LavenderBlush"
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const observatory = {
+  name: "Observatory",
+  image: "assets/telescope.svg",
+  color: "LightSalmon"
+};
+
+const theater = {
+  name: "Theater",
+  image: "assets/doorStar.svg",
+  color: "NavajoWhite"
+};
+
+const guestHouse = {
+  name: "Guest House",
+  image: "assets/realEstate.svg",
+  color: "PeachPuff"
+};
+
+const patio = {
+  name: "Patio",
+  image: "assets/outdoor.svg",
+  color: "Olive"
+};
 
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard
+];
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
 
-const rooms = []
+const rooms = [
+  diningRoom, 
+  conservatory,
+  kitchen,
+  study, 
+  library, 
+  billiardRoom, 
+  lounge, 
+  ballroom,
+  hall, 
+  spa,
+  livingRoom, 
+  observatory, 
+  theater, 
+  guestHouse, 
+  patio
+];
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+const mystery = {
+  killer: undefined,
+  weapon: undefined,
+  room: undefined 
+};
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// FUNCTIONS
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  mystery.killer = randomSelector(suspects);
+
+  const killerImage = document.getElementById("killerImage");
+  const favouriteWeapon = document.getElementById("favouriteWeapon");
+  const killerAge = document.getElementById("killerAge");
+  const killerName = document.getElementById("killerName");
+
+  killerImage.style.display = "block";
+  favouriteWeapon.style.display = "block";
+  killerAge.style.display = "block";
+  killerName.style.display = "block";
+  killerCard.style.background = mystery.killer.color;
+  document.getElementById("loadingAnimation1").style.display = "none";
```

Comment:
`loadingAnimation1` could be something more specific

 `document.getElementById("loadingAnimation1").style.display = "none";` 
This line of code by itself would be very difficult to understand in relation to what will change on the page